lib/src/mockserver/server.ts

```diff
@@ -29,6 +29,12 @@ export function runMockServer(
 ) {
   const app = express();
   app.use(cors());
+  app.use((req, resp, next) => {
+    if (req.path.includes("/_draft/")) {
+      req.url = stripDraft(req.url);
```

Comment:
As discussed offline - was wondering what was the point of just stripping the `_draft` prefix without anything particular with it. @fwouts should we also add a context to the request, something along the lines of `req.isDraft = true`, and return mock data based on it ?

Reply:
a good discussion to start with @fwouts

Reply:
it's mostly because the request coming in the server does not have a concept of `isDraft` we can only know that a request is a draft from it and without stripping that out, we cannot get the endpoint that we want to correctly mock.

Reply:
just to clarify, the `/_draft/` prefix is also for developers to know when they're debugging which endpoints are returning mock responses.